modules/installation-mirror-repository.adoc

```diff
@@ -34,7 +34,7 @@ $ export LOCAL_SECRET_JSON='<path_to_pull_secret>' <5>
 $ export RELEASE_NAME="ocp-release" <6>
 ----
 <1> For `<release_version>`, specify the version number of {product-title} to
-install, such as `4.3.0`.
+install, such as `4.3.0-x86_64`.
```

Comment:
Unless you're installing for another arch?

Reply:
@wking, I'm not aware that another arch is currently available for 4.3. Is the syntax for Z going to be `4.3.x-s390x`?

Reply:
@wking, who should I check with about suffixes for other arches?

Reply:
Talk to ART about suffixes for other arches and official release timelines.  But personally I think we want #19266 so we don't need to care about what ART decides to use for tag names ;).